index.bs

```diff
@@ -400,6 +438,16 @@ spec:url; type:dfn; text:scheme
     </wpt>
   </section>
 
+  <section algorithm="htmlportalelement-may-have-guest-browsing-context">
+    To determine whether a <{portal}> element <dfn for="HTMLPortalElement">may have a guest browsing context</dfn>, run the following steps:
+
+    1. If |element| is [=browsing-context connected=], then return true.
```

Comment:
Maybe also add "and its browsing context is top-level"?

Reply:
Done; PTAL?